apm-webapp/pom.xml

```diff
@@ -29,18 +30,18 @@
     <packaging>jar</packaging>
 
     <properties>
+        <java.version>1.8</java.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <spring.boot.version>1.5.22.RELEASE</spring.boot.version>
+        <spring.boot.version>2.4.8</spring.boot.version>
         <log4j.version>2.6.2</log4j.version>
         <gson.version>2.8.2</gson.version>
         <apache-httpclient.version>4.5.3</apache-httpclient.version>
-        <spring-cloud-dependencies.version>Edgware.SR1</spring-cloud-dependencies.version>
+        <spring-cloud-dependencies.version>2020.0.3</spring-cloud-dependencies.version>
         <frontend-maven-plugin.version>1.11.0</frontend-maven-plugin.version>
         <logback-classic.version>1.2.3</logback-classic.version>
         <jackson-version>2.12.2</jackson-version>
         <yaml.version>1.28</yaml.version>
-        <netty.version>4.1.65.Final</netty.version>
-        <tomcat.version>8.5.66</tomcat.version>
+        <tomcat.version>9.0.48</tomcat.version>
```

Comment:
dist/LICENSE should be updated accordingly.

Reply:
I did not found `springboot/cloud` license under `dist/LICENSE`, I'm not sure if I need to add it

Reply:
https://github.com/apache/skywalking/blob/669fe1593c8cd3591652c13d1088c72d2c45ab89/dist-material/release-docs/LICENSE#L254